src/css/styles.css

```diff
@@ -1,48 +1,59 @@
-/* --- IMPORTS --- */
 /* Styles */
 body {
   font: 14px 'Helvetica Neue', Helvetica, Arial, sans-serif;
   background-color: #fff;
   line-height: 1.4em;
   color: #4d4d4d;
-  min-width: 230px;
   max-width: 85%;
   margin: 0 auto;
   -webkit-font-smoothing: antialiased;
   -moz-font-smoothing: antialiased;
   font-smoothing: antialiased;
   font-weight: 300; }
 
-p {
-  color: #38618c; }
+/*p {
+  color: $text-color;
+}
 
 ul {
-  list-style: none; }
+  list-style: none;  
+}
 
 td {
-  padding: 12px; }
+	padding: 12px;
+}
 
 h2 {
-  color: #35a7ff; }
+	color: $secondary-color;
+}
 
 h1 {
-  color: #e7134f; }
+  color: $theme-color;
+}
 
 #consult-banner {
   position: sticky;
   top: 0px;
   margin: 0;
   text-align: center;
-  background-color: #ffe74c;
+  background-color: $ternary-color;
   height: 40px;
-  width: 100%; }
+  width: 100%;
+}
+
+#consult-copy { //TO DO: Vertically align text
+  color: $text-color;
+  font-weight: bold;
+}
 
-#consult-copy {
-  color: #38618c;
-  font-weight: bold; }
 
 #footer-items {
-  background-color: #35a7ff;
-  color: #fff; }
+  background-color: $secondary-color;
+  color: #fff;
+}
+
+#lead {
+  background-color: #777;
+}*/
```

Comment:
Why is all this commented out?

Reply:
Compiling issue between my SCSS and CSS; does not effect the work but it'sa build thing I need to figure out for the future.